### DISCUSSION TITLE
How can the user Cancel or Go Back on a Checkout?

### DISCUSSION THREAD

**Discussion by vacom** [lmsqueezy/lemonsqueezy.js#91] [Q&A] [answered]
Setting up a checkout with the SDK is simple. When a user completes the purchase process, the redirectUrl configuration functions smoothly, redirecting them to the website or app.

However, if the user wishes to cancel or go back for any reason, how can I implement this within the createCheckout method?

Any help is appreciated, thanks!

**Comment by brankoconjic** [comment, marked as answer]
Hi @vacom,

You could either use a checkout overlay:
https://www.lemonsqueezy.com/ecommerce/checkout-overlays

or open a hosted checkout in a new tab.

Currently, there isn't a way to include a back button in a hosted checkout (which is probably better for conversion rates).

**Reply by vacom** [reply]
Hey @brankoconjic, that makes sense, It is better than nothing will try that experience

What about this question https://github.com/lmsqueezy/lemonsqueezy.js/discussions/90 that also opened, this one makes sense to have it on the API/SDK, right?
